it_hardware/data/knowledge_article.xml

```diff
@@ -0,0 +1,176 @@
+                    <strong>Confirm Sales Order</strong>
+                </p>
+                <p style="margin-bottom: 0px;">As the customer confirms the sales order, one task will be created in the 'Installation Project'(Existing).</p>
+                <p style="margin-bottom: 0px;">
+                    <br/>
+                </p>
+                <p style="margin-bottom: 0px;">
+                    <strong>Deliver products</strong>
+                </p>
+                <p style="margin-bottom: 0px;">You can deliver the 1 product("CCTV Camera") from the inventory application. Use the barcode scanner, or the backend, up to you.</p>
+                <p style="margin-bottom: 0px;">Warranty date for above product will be calculated from the date of delivery.(<font class="text-o-color-1">Through Studio, Automated Action-Update Warranty Date</font>)</p>
+                <blockquote data-o-mail-quote="1" data-o-mail-quote-node="1" style="margin-bottom: 0px;">For the Some Business warranty will start from date of the delivery.</blockquote>
+                <p style="margin-bottom: 0px;">
+                    <br/>
+                </p>
+                <p style="margin-bottom: 0px;">
+                    <strong>Installation - CCTV Camera</strong>
+                </p>
+                <p style="margin-bottom: 0px;">Assign the field service engineer on the task of "<font class="text-o-color-1">Installation Service</font>".</p>
+                <p style="margin-bottom: 0px;">
+                    After the planning, field service engineer will visit the client place and start working on installation, they move the task "<font class="text-o-color-1">In Progress</font>" and then "<font class="text-o-color-1">Done</font>" when it's finished and sign the service report.
+                </p>
+                <p style="margin-bottom: 0px;">
+                    <br/>
+                </p>
+                <h3 style="margin-bottom: 0px;">Flow 4: Repair Service</h3>
+                <p style="margin-bottom: 0px;">If the product are in warranty and customer has any complaint which need to repair the service. we can repair the product and generate the Invoice if required.</p>
+                <p style="margin-bottom: 0px;">
+                    <br/>
+                </p>
+                <p style="margin-bottom: 0px;">To do that,</p>
+                <ul style="margin-bottom: 0px;">
+                    <li>Go to website and Helpdesk form</li>
+                    <li>when open the form choose product ('<font class="text-o-color-1">ThinkPad X1 Carbon 35.56cms - 12th Gen Intel i7</font>') and serial number('CNC012345') and submit ticket.</li>
+                    <li>support ticket will be created from backend.</li>
+                    <li>Open the ticket from backend, system will be displayed the Warranty Date('<font class="text-o-color-1">24/08/2024</font>') of that selected serial number(<font class="text-o-color-1">through Studio</font>).</li>
```

Comment:
```suggestion
                    <li>Open the ticket from backend, system will be displayed the Warranty Date of that selected serial number.</li>
```

Reply:
That can be automated but we need a server action for this

Reply:
So this doesn't work right now. We either need to remove that sentence or to add the action

Reply:
We still need to fix this